pandas/core/frame.py

```diff
@@ -1900,6 +1901,13 @@ def to_dict(
             instance of the mapping type you want.  If you want a
             collections.defaultdict, you must pass it initialized.
 
+        index : bool, default True
+            Whether to include the index item (and index_names item if `orient`
+            is 'tight') in the returned dictionary. Can only be ``False``
+            when `orient` is 'split' or 'tight'.
+
+            .. versionadded:: 1.5.0
```

Comment:
This will be added in 1.6.0 (probably 2.0). Can you change this to 1.6 and move the whatsnew note please?

Reply:
Done.